src/types.ts

```diff
@@ -106,9 +111,39 @@ const asFiatPaymentType = asValue(
 export type FiatPaymentType = ReturnType<typeof asFiatPaymentType>
 
 /** The type of exchange that the partner is. A 'fiat' type means on/off ramp. */
+export type ExchangeType = ReturnType<typeof asExchangeType>
 const asExchangeType = asValue('fiat', 'swap')
 
-export const asStandardTx = asObject({
+export interface StandardTx {
+  orderId: string
+  countryCode: string | null
+  depositTxid?: string
+  depositAddress?: string
+  depositCurrency: string
+  depositAmount: SafeNumber
+  direction: Direction
+  exchangeType: ExchangeType
+  paymentType: FiatPaymentType | null
+  payoutTxid?: string
+  payoutAddress?: string
+  payoutCurrency: string
+  payoutAmount: SafeNumber
+  status: Status
+  usdValue: number
+
+  /** When the transaction occurred (ISO date). */
+  isoDate: string
+  /** When the transaction occurred (unix timestamp). */
+  timestamp: number
+
+  /** When the document was created. */
+  createTime?: Date
```

Comment:
This should be called `updateTime` since it's not when the doc was created but when it was last updated. the reports engine will continually update the doc as it changes status and this would change this time value.